### PR TITLE
Issue 5

### PR DESCRIPTION
**Pull request by kai-li-27**
Closes #X

---

The following is a checklist to remind you of the steps to take when opening a
new PR. The horizontal line above and everything below can be removed from the
pull request description.

- [ ] I have replaced `X` above with the associated issue number.
- [ ] I have changed the issue's label to "pull request".
- [ ] I have assigned the designated reviewer to this PR.
- [ ] I have set the appropriate label on this PR.
- [ ] I have written clear commit messages ([ref](http://chris.beams.io/posts/git-commit/)).

If needed, please review the document on the
[Image Sharing Project](https://sites.google.com/a/appfolio.com/eng/new-hire-resources/engineering-academy/ropes/image-sharing-project)
for information on how labels should be used.

### CODE REVIEW

**Comment by kai-li-27** [comment]
[Issue 5: Image Tags](https://trello.com/c/CGbJlVAu/162-issue-5-image-tags)

File: .rubocop.yml [hunk 1]
```diff
@@ -6,6 +6,11 @@ AllCops:
     - 'vendor/**/*'
     - 'node_modules/**/*'
     - 'bin/*'
+    - 'db/migrate/20190703231910_add_missing_indexes_on_taggings.acts_as_taggable_on_engine.rb'
```
Comment (fanniey) [inline .rubocop.yml]: The gem name is actually `acts-as-taggable-on`.

File: test/controllers/images_controller_test.rb [hunk 2]
```diff
@@ -68,4 +68,12 @@ def test_new_has_tag_input
...
     assert_select 'input[name="image[tag_list]"]'
   end
+
+  def test_show_has_tag
+    image = Image.create!(url: 'https://a', tag_list: 'tag')
+
+    get image_path(image)
+
+    assert_select 'label.badge', 'tag'
+  end
```
Comment (fanniey) [inline test/controllers/images_controller_test.rb]: Can we also make sure it render correctly when there is no tag?

File: test/controllers/images_controller_test.rb [hunk 3]
```diff
@@ -68,4 +68,12 @@ def test_new_has_tag_input
 
     assert_select 'input[name="image[tag_list]"]'
   end
+
+  def test_show_has_tag
+    image = Image.create!(url: 'https://a', tag_list: 'tag')
+
+    get image_path(image)
+
+    assert_select 'label.badge', 'tag'
```
Comment (fanniey) [inline test/controllers/images_controller_test.rb]: Can we also make sure that we only find 1 tag only?

File: test/controllers/images_controller_test.rb [hunk 4]
```diff
@@ -76,4 +76,14 @@ def test_show_has_tag
...
   end
+
+  def test_index_has_tags
+    Image.create!(url: 'https://a', tag_list: 'tag1')
+    Image.create!(url: 'https://a', tag_list: 'tag2')
+
+    get images_path
+
+    assert_select 'label.badge', 'tag1'
+    assert_select 'label.badge', 'tag2'
```
Comment (fanniey) [inline test/controllers/images_controller_test.rb]: What you are doing here is selecting from the entire page regardless of the sequence. Can we make sure that different images have the correct tags on them.

**Pull request merged**